src/System.Text.Primitives/System/Text/Parsing/PrimitiveParser2_bool.cs

```diff
@@ -10,6 +10,54 @@ public static partial class PrimitiveParser
     {
         #region helpers
 
+        private unsafe static bool IsTrue(byte* utf8Bytes, int length)
+        {
+            if (length < 4)
+                return false;
+
+            byte firstChar = utf8Bytes[0];
+            if (firstChar != 't' && firstChar != 'T')
```

Comment:
https://github.com/dotnet/corefxlab/issues/963

^ I have created an issue and assigned myself.